.chloggen/jsruntime-metrics.yaml

```diff
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: new_component
+
+# The name of the area of concern in the attributes-registry, (e.g. http, cloud, db)
+component: jsruntime
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Introducing semantic conventions for Runtime Javascript (e.g. NodeJS, DenoJS, BunJS) runtime metrics.
```

Comment:
I don't believe Deno and Bun use the JS suffix

Reply:
removed